build.md

```diff
@@ -0,0 +1,41 @@
+# Build luet packages for kairos
+
+kairos uses github action to build packages and push it to quay.io/kairos/packages(amd64 version) or quay.io/kairos/packages-arm64 (arm64 version),
+in case you wanna build it manually, e.g. testing a new added package locally, or create your own repo to serve it offline/privately, it can be done with this steps:
+
+## build packages
+
+To build all packages, go repo root dir, and issue commands like this (note, you need to run as root user, otherwise you will get some permission issues):
```

Comment:
```suggestion
To build all packages, go to the repo root dir, and issue commands like this (note, you need to run as root user, otherwise you will get some permission issues):
```

Reply:
fixed